functions-homework.js

```diff
@@ -0,0 +1,46 @@
+// 1
+function getGreetings(name) {
+  alert('Hi, ' + name + ' What`s up?');
+}
+//getGreetings('kuba klin');
+
+// 2
+function isThisMyName(personName) {
+  if (personName === 'Kuba') {
+    return true;
+  }
+  return false;
+}
+console.log(isThisMyName('Kuba'));
+
+// 3
+function isThisBoolean(testValue) {
+  if (typeof testValue == 'boolean') return true;
```

Comment:
Let's use the knowledge from the lesson about logical operators. You could solve this function in an elegant way in just one line with a single return statement.